process/mock/txProcessorMock.go

```diff
@@ -31,8 +31,5 @@ func (etm *TxProcessorMock) ProcessSmartContractResult(scr *smartContractResult.
 
 // IsInterfaceNil returns true if there is no value under the interface
 func (etm *TxProcessorMock) IsInterfaceNil() bool {
```

Comment:
please rename the receiver for all methods in this mock

Reply:
done.